requirements.txt

```diff
@@ -1,5 +1,5 @@
 arrow>=1.0.0
-click>=7.0,<8.0
+click>=8.0,<9.0
```

Comment:
I don't think we should explicitly forbid 9.0, we don't know if that will be a breaking change for us yet. We usually try to avoid pinning versions if we can.

Reply:
I restricted to 8 partly because it was a "MAJOR" version change (7 to 8) that broke Watson. I'm not familiar with either project, but shouldn't major releases of dependencies only be (explicitly) updated to once they've been tested due to the likelihood of breaking changes?

Reply:
This is a long debate in the Python community :) . So far my opinion is to allow any version of a dependency as long as you don't know for sure that you're not compatible with it. This is less of an issue if you isolate the dependencies of each Python project you're using, but if you're not doing it things can get messy pretty quickly if everybody is asking for specific versions of a package for no proper reason.

Reply:
Huh - interesting! Seems like asking for trouble to me. :D Will you remove the <9 requirement or should I?

Reply:
Hi David! Can you fix this, and then I'll merge your work and release it. Thanks!

Reply:
Thanks! Done.